src/main/kotlin/me/partlysanestudios/partlysaneskies/utils/StringUtils.kt

```diff
@@ -407,4 +408,14 @@ object StringUtils {
         }
     }
 
+    fun String.lastUsedColorCode(): String? {
```

Comment:
could we get start and end indexes?
```suggestion
    fun String.lastUsedColorCode(startIndex: Int = 0, endIndex: Int = this.length - 1): String? {
```

I believe that syntax works?